bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpRendererHandler.java

```diff
@@ -1113,6 +1113,11 @@ private void onValueReceivedVolume(String variable, @Nullable String value) {
             long volume = Long.valueOf(value);
```

Comment:
Now we're at it (adding sanity check), is there any risk of receiving something which is not a numbe? This would throw `NumberFormatException`.

Reply:
According to the uPnP standard, volume is defined as a unsigned integer between 0 and 100 (https://upnp.org/specs/av/UPnP-av-RenderingControl-v1-Service.pdf#page15). Obviously, we can check for this all the time, but I don’t want to overload these checks to police bad implementations everywhere. In that case, we should do a sanity check everywhere we receive something (not just volume) and log a warning when the device does not comply.

Reply:
OK, clearly outside the scope of this PR.